app/frontend/styles/application.css

```diff
@@ -0,0 +1,3 @@
+@import './tailwindcss/base.css';
```

Comment:
This is the way [tailwind should be used](https://tailwindcss.com/docs/installation#2-add-tailwind-to-your-css) when we need to import more css files (in the next homepage redesign PR).